lib/plugins/pumpiob.js

```diff
@@ -0,0 +1,71 @@
+/* jshint node: true */
+'use strict';
+
+var _ = require('lodash');
+
+function init() {
+
+  var pumpiob = {
+    name: 'pumpiob'
+    , label: 'Pump IOB'
+    , pluginType: 'pill-major'
+  };
+
+  pumpiob.setProperties = function setProperties(sbx) {
+    sbx.offerProperty('pumpiob', function setPumpIOB () {
```

Comment:
I was hoping we'd be able to use the same `iob` property here, then any other plugins like BWP could get the info they need.